pages/functional/js/fibonacci.js

```diff
@@ -1,86 +1,42 @@
 import { parseNumber } from "../../js/helper.js";
 
 /**
- * Create an array filled by fibonacci sequence using loop way.
- *
- * @param {number} sequence
- */
-function generateFibonacciUsingLoop(sequence) {
-  sequence = parseNumber(sequence);
-
-  let result = [];
-  let prev = 0;
-  let next = 1;
-
-  for (let index = 0; index < sequence; index++) {
-    result.push(prev);
-
-    const current = prev + next;
-    prev = next;
-    next = current;
-  }
-
-  return result;
+* Using Recursive
+* @param {number} sequence 
+* @returns 
+*/
+function fibbonaciUsingRecursive (sequence) {
+  return sequence < 1
+  ? 0 //cancelling execution for unreal number
+  : sequence <= 2
+  ? 1
+  : fibbonaciUsingRecursive (sequence - 1) +
+    fibbonaciUsingRecursive (sequence - 2);
 }
 
 /**
- * Create an array filled by fibonacci sequence using recursive way.
- *
- * @param {number} sequence
- */
-function generateFibonacciUsingRecursive(sequence) {
-  /**
-   * @param {number} n
-   * @returns {number}
-   */
-  function fibonacci(n) {
-    return n < 1 ? 0 : n <= 2 ? 1 : fibonacci(n - 1) + fibonacci(n - 2);
+* Using Loop
+* @param {number} sequence 
+* @returns 
+*/
+function fibbonaciUsingLoop (sequence) {
+  if (sequence < 1) {
+      return 0;
   }
-
-  sequence = parseNumber(sequence);
-
-  let result = [];
-
-  for (let index = 0; index < sequence; index++) {
-    const current = fibonacci(index);
-
-    result.push(current);
+  else if (sequence <= 2) {
+      return 1
   }
+  let previous = 0;
+  let next = 1;
+  let result = 1;
```

Comment:
Give a break for each part of your logic code for a better reading purpose, for example on this. You need to separate the declaration variable from the other part by giving a line break.